model/build.gradle.kts

```diff
@@ -2,14 +2,20 @@ plugins {
 	id("kotlin")
 }
 
+dependencies {
+	implementation(kotlin("stdlib-jdk7"))
```

Comment:
Why not stdlib-jdk8?

Reply:
I think I forgot to change it in this specific file. Will be fixed in another PR as I want to migrate everything to buildSrc